<h1 style='font-size:40px'> S3 Security</h1>

<h2 style='font-size:30px'> S3 Lifecycle Rules (with S3 Analytics)</h2>
<div> 
    <ul style='font-size:20px'> 
        <li> 
        </li>
    </ul>
</div>

In [2]:
! git add .
! git commit -am 'Aula 142'
! git push

[master 477162f] Aula 141
 4 files changed, 276 insertions(+), 24 deletions(-)
 create mode 100644 img/13_byte_range.png
 create mode 100644 img/13_s3_event.png
Enumerating objects: 11, done.
Counting objects: 100% (11/11), done.
Delta compression using up to 24 threads
Compressing objects: 100% (7/7), done.
Writing objects: 100% (7/7), 88.37 KiB | 29.46 MiB/s, done.
Total 7 (delta 3), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (3/3), completed with 3 local objects.
To https://github.com/felipesveiga/aws-developer.git
   49afaba..477162f  master -> master


<p style='color:red'> Vi Aula 141; Pesquisar sobre chaves de encriptação; Aula 142</p>